python27_sti_base/Makefile

```diff
@@ -0,0 +1,46 @@
+BASE_IMAGE=centos/python-27-centos7:latest
+LOCAL_IMAGE=openshift-spark-py27i-sti
+SPARK_IMAGE=myproject/openshift-spark-py27-sti
+
+.PHONY: build clean push create destroy artifacts clean-artifacts clean-dockerfile clean-scripts clean-template
+
+build:
+	docker build -t $(LOCAL_IMAGE) .
```

Comment:
well, I broke the targets apart so the conflict went away.  I had the "artifacts" as deps for build, but this caused the copy (and hence file creation) to run as root when sudo was supplied outside.

So now, if you run build without artifacts, build fails because there is no Dockerfile.  Seemed like a decent compromise.  It's probably possible to break apart similar targets in other Makes and follow this pattern.